gke/variables.tf

```diff
@@ -21,6 +21,10 @@ variable "node_count" {
   default = "3"
 }
 
+variable "preemptible" {
+    default = false
```

Comment:
Do we also need to say `type: bool` ?

Reply:
I would indeed add type = bool to the variable definition, to do things right.
https://www.terraform.io/docs/providers/google/r/container_cluster.html#preemptible